streams/test-utils/src/main/java/org/apache/kafka/streams/TopologyTestDriver.java

```diff
@@ -45,7 +45,6 @@
 import org.apache.kafka.streams.internals.WindowStoreFacade;
 import org.apache.kafka.streams.processor.ProcessorContext;
 import org.apache.kafka.streams.processor.PunctuationType;
-import org.apache.kafka.streams.processor.Punctuator;
```

Comment:
Why this change? Importing simplifies JavaDocs.

Reply:
It does, but it does not work well with "Organize imports". Which I usually do to avoided errors for unused imports.
The problem here is that Punctuator is never used in a javadoc link, therefore it is not recognized as a required import, at least not in my IDE.
So using the fully qualified name there seemed safer.
Any suggestion is welcome :-)

Reply:
Seems to be shortcoming of your IDE. I don't have issue with IntelliJ. Maybe it's best to not do "Organize imports" and rely on the gradle checkstyle that will detect unused imports anyway.?

Reply:
Ok, I have reverted this part.